cmd/repo.go

```diff
@@ -303,13 +311,12 @@ func getFilesNotIndexed(index repo.Index, distributedFiles string) []os.FileInfo
 
 	for _, artifact := range index {
 		if artifact.Installed {
-			installedInIndex[artifact.FilePath(distributedFiles)] = true
 			installedInIndex[filepath.Join(distributedFiles, "all", artifact.Filename)] = true
 		}
 	}
 
 	filesNotIndexed := make([]os.FileInfo, 0)
-	err := filepath.Walk(distributedFiles,
+	err := filepath.Walk(filepath.Join(distributedFiles, "all"),
```

Comment:
Why do you not use a simple filepath.Glob (https://golang.org/pkg/path/filepath/#Glob)?

Reply:
Because filepath.Glob check only files inside `/all` without walk any directory inside it.
But if we don't want that there's any directory inside it, it work good.

Reply:
Because it doesn't walk inside directories, but if it is what we want it's okay.

Reply:
I think it's a good idea

Reply:
Ok, I'm going to add a WARN log for any directory I found, but I won't add files within!